src/style.css

```diff
@@ -1 +1,25 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
no need the last star of every row to have margin, please add some selector no avoid margin-right on the last star :wink: